contracts/upgrades/DFArenaGetterFacet2.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Why do we need ArenaGetterFacet2?

Reply:
`getArenaConstants` grants access to new game constants like modifiers and anything else we add before our next backend deploy